### PR TITLE
PLATSUP-1013_platform_docs_updates

### PR DESCRIPTION
**Pull request by marinahand**
misc updates from feedback in Slack

### CODE REVIEW

**Comment by justin-lee-zapier** [comment]
When you're ready for @team-platform-support to review this, please head over to this [Slack thread](https://zapier.slack.com/archives/C02PCUNBUG7/p1706561299914989) and give us a `:review-pr:` emoji.

File: docs/_build/fields/field-definitions.md [hunk 1]
```diff
@@ -71,4 +71,4 @@ Dynamic fields make a request to your app's API, showing the returned data in a
 
 ## Line item groups
 
-Line item groups allows for a set of input fields to create line items. When a comma separated list is entered in the field, each list item would create a new separate object in your app. Typically used to add multiple lines of similar details to an app, for example in invoice and accounting apps. Only available in Platform UI actions. 
\ No newline at end of file
+Line item groups allow users to input line items provided from the trigger or prior action. Those items will then be sent in the request as an array of individual objects. Users cannot input comma separated values into a line item field, they need to be provided from the trigger or a prior action in line item format for them to be sent this way. Typically used to add multiple lines of similar details to an app, for example in invoice and accounting apps. Only available in Platform UI actions.
```
Comment (redbeardaz) [inline docs/_build/fields/field-definitions.md:74]: >Only available in Platform UI actions.

I know this was already there, but does this make sense to devs? Maybe link to this article (https://platform.zapier.com/quickstart/ui-vs-cli) and/or explicitly say not supported via CLI?

Comment (marinahand) [reply docs/_build/fields/field-definitions.md:74]: Sounds good, updated to link to that article, could you re-review? Thank you 😊

**Review by justin-lee-zapier — APPROVED** [review]
Looks good, the new link works for me.

**Pull request merged**